src/Reflection/ReflectionProperty.php

```diff
@@ -233,7 +233,7 @@ public function getImplementingClass() : ReflectionClass
 
     public function getDocComment() : string
     {
-        return GetFirstDocComment::forNode($this->node);
+        return GetLastDocComment::forNode($this->node);
```

Comment:
The modifications on the various `Reflection*` API endpoints need test additions here

Reply:
Added tests.